src/Compilers/Core/Portable/Symbols/WellKnownMemberNames.cs

```diff
@@ -298,5 +298,11 @@ public static class WellKnownMemberNames
         /// (see C# Specification, §7.7.7.1 Awaitable expressions).
         /// </summary>
         public const string OnCompleted = nameof(OnCompleted);
+
+        /// <summary>
+        /// The required name for the <c>CreateAsyncMethodBuilder</c> method used to
+        /// build a tasklike instance.
+        /// </summary>
+        public const string CreateAsyncMethodBuilder = "CreateAsyncMethodBuilder";
```

Comment:
> public const string CreateAsyncMethodBuilder [](start = 7, length = 45)

WRT my earlier comment about being public: looks like this is the sttandard pattern we follow here.